modules/apps/style-book/style-book-test/src/testIntegration/java/com/liferay/style/book/util/test/DefaultStyleBookEntryUtilTest.java

```diff
@@ -75,6 +79,7 @@ public void testGetDefaultMasterStyleBookEntry() throws Exception {
 			DefaultStyleBookEntryUtil.getDefaultMasterStyleBookEntry(
 				masterLayoutBasedLayout);
 
+		Assert.assertNotNull(defaultMasterStyleBookEntry);
```

Comment:
Is this assertion needed since in the next assertion we call `defaultMasterStyleBookEntry.getStyleBookEntryId()` which would throw an error? Also if we need this assertion to come before the next assertion, should we add a line break to indicate that it shouldn't be sorted? Same for the one below.